Easydict/Swift/Utility/GlobalContext.swift

```diff
@@ -21,6 +21,12 @@ class GlobalContext: NSObject {
             updaterDelegate: updaterHelper,
             userDriverDelegate: userDriverHelper
         )
+
+        for service in services {
```

Comment:
Could this logic able to move inside `StreamConfigurationView` instead?

Reply:
No, we need to ensure that the LLM service is only subscribed once and remains unreleased, and it is appropriate to put it in the initialization method of this singleton.

Reply:
OK, this sounds reasonable.

I just concern the 'GlobalContext' sounds not the right place to do this. Maybe we could improve this in the future.

Reply:
Yes, this code should be improved later.